core/src/com/unciv/ui/newgamescreen/NewGameScreen.kt

```diff
@@ -293,7 +295,19 @@ class NewGameScreen(
             }
         }
 
-        val worldScreen = game.loadGame(newGame)
+        val worldScreen: WorldScreen
+        try {
+            worldScreen = game.loadGame(newGame)
+        } catch (ex: Exception) {
+            val (message) = LoadGameScreen.getLoadExceptionMessage(ex)
+            launchOnGLThread {
+                popup.reuseWith(message, true)
+                rightSideButton.enable()
+                rightSideButton.setText("Start game!".tr())
+            }
+            Gdx.input.inputProcessor = stage
+            return@coroutineScope
+        }
```

Comment:
Waitasecond... you can create a game that you can't load? That sounds like a mistake.
Maybe if someone tries to create a MP game and his ID isn't in there then warn him?